ui/scripts/ui-custom/routingPolicyWizard.js

```diff
@@ -0,0 +1,489 @@
+(function($, cloudStack) {
```

Comment:
a lot of files without licenses in the old ui which is no longer part of ACS, best to just delete them all from this PR (i.e. `ui/scripts/*`

Reply:
i deleted it